crates/sui-indexer/src/schema.rs

```diff
@@ -71,13 +71,24 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    input_objects (id) {
+        id -> Int8,
```

Comment:
why is this `Int8` but the struct definition is `u64`?

```
pub struct InputObject {
    pub id: Option<i64>,
    pub transaction_digest: String,
    pub checkpoint_sequence_number: i64,
    pub epoch: i64,
    pub object_id: String,
    pub object_version: Option<i64>,
}
```

Reply:
Int8 means 8 bytes, which is 64 bits;  and PG does not support unsigned number type, so u64 is converted to int8.